### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by jon-schlandt** [jon-schlandt/curiosity-curator#38]
The current lighthouse score for this application is 81. This should be improved to near perfect if possible. Additional testing needs to be done on the Favorites view and through other accessibility tools like WAVE.
